frame/fnft/src/test/nonfungibles/inspect_enumerable.rs

```diff
@@ -0,0 +1,96 @@
+use frame_support::traits::tokens::nonfungibles::{Create, InspectEnumerable};
+use std::collections::BTreeSet;
+
+use crate::test::{
+	mock::{new_test_ext, Nft},
+	prelude::*,
+	ALICE, BOB,
+};
+
+#[test]
+pub(crate) fn test() {
+	new_test_ext().execute_with(|| {
+		let first_collection = 1_u16;
+		let second_collection = 2_u16;
+		Nft::create_collection(&first_collection, &ALICE, &BOB).unwrap();
+		Nft::create_collection(&second_collection, &BOB, &ALICE).unwrap();
+		let first_collection_items_alice = mint_many_nfts_and_assert::<3>(ALICE, first_collection);
+		let second_collection_items_alice =
+			mint_many_nfts_and_assert::<6>(ALICE, second_collection);
+		let first_collection_items_bob = mint_many_nfts_and_assert::<20>(BOB, first_collection);
+		let second_collection_items_bob = mint_many_nfts_and_assert::<1>(BOB, second_collection);
+
+		assert_eq!(
+			<Nft as InspectEnumerable<u128>>::collections().collect::<BTreeSet<_>>(),
+			BTreeSet::from([first_collection, second_collection])
+		);
+
+		assert_eq!(
+			<Nft as InspectEnumerable<u128>>::items(&first_collection).collect::<BTreeSet<_>>(),
+			BTreeSet::from_iter(
+				first_collection_items_alice
+					.into_iter()
+					.chain(first_collection_items_bob.into_iter())
+			)
```

Comment:
```suggestion
				first_collection_items_alice
					.into_iter()
					.chain(first_collection_items_bob)
					.collect()
```

Reply:
Just a nit, I personally find this more readable but it's just a stylistic thing

Reply:
It does look better indeed :)